sdk/resourcemanager/azure-resourcemanager-authorization/src/main/java/com/azure/resourcemanager/authorization/implementation/PermissionImpl.java

```diff
@@ -26,7 +26,7 @@ public List<String> notActions() {
 
     @Override
     public List<String> dataActions() {
-        return innerModel().actions();
+        return innerModel().dataActions();
     }
 
     @Override
```

Comment:
Is "notDataActions" below also not correct?

Reply:
They're not correct. Fixed as well. Thanks!